packages/ripple/README.md

```diff
@@ -0,0 +1,54 @@
+# Ripple
```

Comment:
We need documentation somewhere that the element needs `overflow: hidden;`

Reply:
👍